.github/workflows/keyvi.yml

```diff
@@ -0,0 +1,83 @@
+# Build on Linux
+
+name: Build keyvi
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ${{matrix.os}}
+    name:  ${{ matrix.type }} on ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        type: ['Release', 'Debug']
+        os: ['macos-latest', 'ubuntu-latest']
+    steps:
+      - name: install Linux deps
+        if: runner.os == 'Linux'
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y libsnappy-dev libzzip-dev zlib1g-dev libboost-all-dev
+          sudo apt-get install ccache
+      - name: install macOS deps
+        if: runner.os == 'macOS'
+        run: |
+          brew update
+          brew install zlib snappy openssl gnu-getopt coreutils boost
```

Comment:
Why do we install `openssl gnu-getopt coreutils` here ? 
I don't remember us doing it before.

Reply:
not sure, I took this over from the old definitions afair.

Will give it a try.